doc/ceph-volume/lvm/encryption.rst

```diff
@@ -4,42 +4,43 @@ Encryption
 ==========
 
 Logical volumes can be encrypted using ``dmcrypt`` by specifying the
-``--dmcrypt`` flag when creating OSDs. Encryption can be done in different ways,
-specially with LVM. ``ceph-volume`` is somewhat opinionated with the way it
-sets up encryption with logical volumes so that the process is consistent and
+``--dmcrypt`` flag when creating OSDs. When using LVM, logical volumes can be
+encrypted in different ways. ``ceph-volume`` does not offer as many options as
+LVM does, but it encrypts logical volumes in a way that  is consistent and
 robust.
 
 In this case, ``ceph-volume lvm`` follows these constraints:
 
-* Logical Volumes are encrypted, while their underlying PVs (physical volumes)
-  aren't
-* Non-LVM devices like partitions are also encrypted with the same OSD key
+* Logical Volumes (LVs) are encrypted, but their underlying physical volumes
+  (PVs) are not encrypted.
```

Comment:
I haven't worked a lot with `ceph-volume` but this is surprising and alarming.  Encryption-at-rest is the whole point of `--dmcrypt`.  I suspect though that the wording here is misleading, that what it means to say is that encryption is applied at the LVM level, and that the encrypted data is written to the underlying physical *devices*, vs using something like OPAL or SED.  I suggest checking with Neha or an OSD/RADOS authority for clarification here.

Reply:
I think this is strong enough to be a blocker. @neha-ojha: is it true that the physical volumes that support logical volumes aren't encrypted when you use ``--dmcrypt``?

Reply:
Ack.  Really I don't know how they couldn't be, I strongly suspect this is misleading wording.

Reply:
I've spoken to Josh about it, and we're going to cut this out entirely. It's just confusing.

Reply:
Agreed.